src/components/notification/AposNotification.vue

```diff
@@ -111,9 +149,38 @@ export default {
     height: 12px;
   }
 
-  .apos-notification__label {
+  .apos-notification__label,
+  .apos-notification__progress-value {
     font-size: map-get($font-sizes, default);
+  }
+
+  .apos-notification__label {
     letter-spacing: 0.75px;
     color: var(--a-text-primary);
   }
+
+  .apos-notification__progress {
+    display: flex;
+    align-items: center;
+    flex-shrink: 0;
+  }
+
+  .apos-notification__progress-bar {
+    width: 70px;
+    height: 4px;
+    margin-left: 20px;
+    // BG color consistent between themes.
+    background-color: #2c354d;
+    border-width: 0;
+
+  }
+  .apos-notification__progress-now {
+    height: 100%;
+    background-color: var(--a-brand-green);
+    background-image: linear-gradient(46deg, #CC9300 0%, #EA433A 26%, #B327BF 47%, #6666FF 76%, #00BF9A 100%);;
```

Comment:
we have these as vars , `--a-brand-gold` etc

Reply:
nice. I got this from the button component. I'll fix it there as well.